src/main/java/racingcar/domain/Distance.java

```diff
@@ -0,0 +1,17 @@
+package racingcar.domain;
+
+public class Distance {
+    private int distance;
+
```

Comment:
Car 에서 distance 를 0으로 자주 사용하게 된다면 미리 생성해두는것도 좋은 방법이 될 수 있습니다 😄 
(현재는 반영하지 않으셔도 괜찮습니다!!)

```suggestion
public class Distance {
    public static final Distance ZERO = new Distance(0);

    private int distance;

}
```